CMakeLists.txt

```diff
@@ -1,13 +1,14 @@
 cmake_minimum_required(VERSION 3.16)

```

Comment:
this file got autoformatted rather drastically. the modifications in here are mostly to simplify things, removing some extra flags